code/game/world.dm

```diff
@@ -286,7 +286,7 @@ GLOBAL_VAR(restart_counter)
 /world/proc/update_status()
 
 	status = update_status_fulp()  // FULPSTATION Use our code and bail!
-	return
+	/*
```

Comment:
Linter thought this was all dead code, and threw a fit

Reply:
weird but i guess that makes sense, no return was defined even though i knew it would still return